src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/PostgreSqlConnection.php

```diff
@@ -67,7 +67,9 @@ public function get(): ?array
         /** @var \PDO $nativeConnection */
         $nativeConnection = $this->driverConnection->getNativeConnection();
 
-        $notification = $nativeConnection->pgsqlGetNotify(\PDO::FETCH_ASSOC, $this->configuration['get_notify_timeout']);
+        $notification = $nativeConnection instanceof \Pdo\Pgsql
```

Comment:
this will help clean the code on Symfony 9:
```suggestion
        $notification = \PHP_VERSION_ID >= 80500
```